configs/mmfashion/mask_rcnn_r50_fpn_1x.py

```diff
@@ -0,0 +1,201 @@
+# model settings
```

Comment:
You may inherit the basic config files to make this file more clear. （Please refer to https://github.com/open-mmlab/mmdetection/blob/master/configs/mask_rcnn/mask_rcnn_r50_fpn_1x_coco.py)